src/main/java/com/google/cloud/spanner/hibernate/SpannerDialect.java

```diff
@@ -245,6 +250,53 @@ public String getForUpdateSkipLockedString(String aliases) {
         + " acquisition.");
   }
 
+  /* Unsupported Hibernate Exporters */
+
+  @Override
+  public Exporter<Sequence> getSequenceExporter() {
+    return new Exporter<Sequence>() {
```

Comment:
Nice ! I love anonymous classes. 

Since these always just do nothing, would it be better to have  a private static instance ? rather than making new ones on each call ?

Reply:
Good call, done.

Reply:
if you don't give the type param like `Exporter NOOP_SEQUENCE_EXPORTER = new EmptyExporter<>();` you can actually the same instance for all 3 (though you do get a little implicitly-casting warning)

Reply:
Done.